packages/muelu/research/regionMG/src/SetupRegionHierarchy_def.hpp

```diff
@@ -356,7 +356,8 @@ void MakeCoarseCompositeOperator(RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, No
                                        // This estimate is very conservative and probably costs us lots of memory...
                                       8*regMatrix->getCrsGraph()->getNodeMaxNumRowEntries());
   regionalToComposite(regMatrix,
```

Comment:
@pohm01 Feel free to change the interface of `regionalToComposite()`, if this helps to avoid casts.

Reply:
This involves reworking the region unit tests to use const maps which I think I will do as a separate PR